l10n/en/firefox/browsers/compare/opera.ftl

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/compare/opera/
+
+# Page title
+compare-opera-firefox-vs-opera = { -brand-name-firefox } vs Opera
+
+# HTML page description
+compare-opera-read-our-comparison = Read our comparison of { -brand-name-firefox } and Opera browsers on features, privacy, and ease-of-use.
+
+# Variables:
+# $fingerprint (url) - link to https://www.mozilla.org/firefox/features/block-fingerprinting/
+compare-opera-firefoxs-default-privacy-settings = { -brand-name-firefox }’s default privacy settings are stronger than Opera’s – and we have more features to prevent websites from tracking you, such as <a { $fingerprint }>fingerprint blocking</a>.
+compare-opera-firefox-offers-a-wide-range-of = { -brand-name-firefox } offers a wide range of customization options, including the ability to move menus and toolbars to different locations on the browser window. Opera’s interface is less customizable.
+compare-opera-since-we-dont-have-to-make = Since we don’t have to make shareholders happy, we can focus on making <em>you</em> happy and always put your privacy and convenience first.
```

Comment:
**Nit:** This string emphasizes the `you` differently in almost every page:

1. In the Opera and Safari pages, it's italicised
2. In the Chrome page, it's bolded
3. In the Edge page, it has no emphasis
4. (it's not available in the Brave page)

Let's pick which emphasis we want and make it consistent in the pages